### ISSUE TITLE
New Technology Progression

### ISSUE THREAD

**Issue by AtomBlade** [MatEdwards/MIKE#8]
The current tech tree is very disorganized, hard to follow and lacks clear goals for the player to pursue. This causes players to become lose sight of what they are supposed to do and as a result they become confused and quit. 

To alleviate this, I have come up with a new tech tree that focuses on a few clear, tangible goals, that being the creation of increasingly complex electronic circuits to progress in technology. 
Each new electronic circuit unlocked introduces a large amount of new machines and technology, enabling players to do new things that previously were not possible, and it
 should feel very rewarding.

Here is a rough outline of the old technology progression. I have left a lot of things out because it doesn't really have much structure as is, and trying to fill in the whole thing would be way too much work.
**Old Technology tree, read from the bottom up.**
![old progression](https://github.com/MatEdwards/MIKE/assets/13388198/1eda2db8-ff87-4791-b951-d67ead2787ec)

Here is an outline of the new technology progression. It follows a single main trunk where you acquire new materials and circuits, and each major circuit unlocks lots of new technologies.
**New Technology tree, read from the bottom up.**
![new progression](https://github.com/MatEdwards/MIKE/assets/13388198/ea8cbad4-3257-4efc-a01b-65262cfb981e)

Of course the new technology tech tree isn't completely filled in yet, but it should be easy for us to fill in little blanks and put side content along some part of this tech tree.

**Comment by MatEdwards** [comment]
Ignore that reference. That was a mistake.
